packages/assets-controllers/src/TokensController.ts

```diff
@@ -664,7 +694,12 @@ export class TokensController extends BaseController<
       switch (suggestedAssetMeta.type) {
         case 'ERC20':
           const { address, symbol, decimals, image } = suggestedAssetMeta.asset;
-          await this.addToken(address, symbol, decimals, image);
+          /**
+           *
+           * The reason for this undefined is that the wallet_watchAsset doesn't return the name
```

Comment:
Nit: suggested rewording to highlight that this comment is referring to `name`, which was a little bit unclear to me at first
```suggestion
           * The reason for `name` being omitted is that `wallet_watchAsset` doesn't return the name
```